admin-manual/security/security.rst

```diff
@@ -330,9 +330,7 @@ the three environments:
 
 .. code:: bash
 
-   $ sudo /usr/share/archivematica/virtualenvs/archivematica-dashboard/bin/pip install -U certifi
-   $ sudo /usr/share/archivematica/virtualenvs/archivematica-mcp-client/bin/pip install -U certifi
-   $ sudo /usr/share/archivematica/virtualenvs/archivematica-storage-service/bin/pip install -U certifi
+   $ sudo /usr/share/archivematica/virtualenvs/archivematica/bin/pip install -U certifi
```

Comment:
The block above this still says `There are three virtual environments...`, should we update it too?

Reply:
I will fix, thanks!